sdk/core/Azure.Core/src/Shared/StatusCodeClassifier.cs

```diff
@@ -14,34 +12,37 @@ internal class StatusCodeClassifier : ResponseClassifier
     {
         private ulong[] _nonErrors = new ulong[10];
 
-        internal HttpMessageClassifier[]? MessageClassifiers { get; set; }
+        internal HttpMessageClassifier[]? TryClassifiers { get; set; }
 
         public StatusCodeClassifier(int[] nonErrors)
```

Comment:
Since it's an internal type, it might be worth changing this parameter from int[] to ReadOnlySpan<int>. This way callers won't have to allocate GC array to initialize the classifier.

Reply:
Woah: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/stackalloc